examples/README.markdown

```diff
@@ -0,0 +1,24 @@
+# Example Sinatra application
+
+This example illustrated how to configure and work towards Fishbrain identity
+services.
+
+## Getting started
+
+Make sure to have installed the dependencies by running `bundler install`.
+
+```sh
+CLIENT_ID=... CLIENT_SECRET=... bundle exec ruby examples/app.rb
+```
+
+Substituting the ellipses with your credentials.
+
+Visit at [http://localhost:3000](http://localhost:3000).
+
+Note that the example application is configured to run on port `3000` and with a
+path prefix `/client/auth` to the authentication paths.
+
```

Comment:
After logging in to the service, you'll be able to find an identity token (`id_token`) in the page. You can copy this token and restart the server adding `ID_TOKEN=...` (replacing the ellipses accordingly) when re-running the command above.